ndk/src/main/jni/utils/stack_unwinder_libunwind.c

```diff
@@ -1,75 +1,26 @@
 #include "../utils/build.h"
 #include "stack_unwinder_libunwind.h"
+#include <cpu-features.h>
 #include <malloc.h>
 #include <report.h>
 #include <unwind.h>
 
+#if defined(__arm__)
+#include <libunwind.h>
+#endif
+
 typedef struct {
   size_t frame_count;
   uintptr_t frame_addresses[BUGSNAG_FRAMES_MAX];
-#if defined(__arm__)
-  void *signal_context;
-#endif
   siginfo_t *signal_info;
 } bsg_libunwind_state;
 
-bsg_libunwind_state *bsg_global_libunwind_state;
-
-bool bsg_configure_libunwind(void) {
-  bsg_global_libunwind_state = calloc(1, sizeof(bsg_libunwind_state));
-  return true;
-}
+bool bsg_configure_libunwind(void) { return true; }
```

Comment:
is this needed if we're linking the library?

Reply:
All of the unwinders have a config method for consistency.